tests/conftest.py

```diff
@@ -970,12 +974,14 @@ def dvs(request) -> DockerVirtualSwitch:
     keeptb = request.config.getoption("--keeptb")
     imgname = request.config.getoption("--imgname")
     fakeplatform = getattr(request.module, "DVS_FAKE_PLATFORM", None)
-    dvs = DockerVirtualSwitch(name, imgname, keeptb, fakeplatform)
+
+    log_path = name if name else request.module.__name__
+
+    dvs = DockerVirtualSwitch(name, imgname, keeptb, fakeplatform, log_path)
+
     yield dvs
-    if name == None:
-        dvs.get_logs(request.module.__name__)
-    else:
-        dvs.get_logs()
+
+    dvs.get_logs()
```

Comment:
I did not realize we only capture the syslog, we should probably dump the db and capture that as well (not in this pr though)

Reply:
We do also get sairedis.rec and supervisor logs as well, but I think dumping the DB should also help

Reply:
does show tech work in the vs docker? if it works, then we can use that, that captures every thing.

Reply:
It looks like it fails because `sudo` is not found. I'll dig a little deeper and see if we can make it work